### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by 110111-1**
First sprint result 
functional app with one battle mode

### CODE REVIEW

**Review by VincentCMLejeune — DISMISSED** [review]
All set and ready

**Review by Charlotte-Men — DISMISSED** [review]
It's alive !

**Review by isamj27950 — DISMISSED** [review]

File: src/Screens/Fighting results/DisplayFight.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+export default function DisplayFight(props) {
+  const [characterA, setCharacterA] = useState('');
+  const [characterB, setCharacterB] = useState('');
+
+  const { heroA, heroB } = props;
+
+  useEffect(() => {
+    fetch('../../fakeApi.json')
+      .then((res) => res.json())
```
Comment (tyZulh) [inline src/Screens/Fighting results/DisplayFight.jsx:19]: You can make one useEffect for both fetch

**Review by tyZulh — CHANGES_REQUESTED** [review]
There is some changes to be made : 
-remove the dead code from your css
-check the import order in your components
-combine your fetch in one unique useEffect

To make  these changes  pull a new branches from dev

**Review by pelegrand — APPROVED** [review]
ok

**Pull request merged**